.github/workflows/codeql-analysis.yml

```
@@ -1,13 +1,13 @@
-name: "CodeQL"
+name: 'CodeQL'
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)